src/constants/npc_constants.asm

```diff
@@ -1,3 +1,38 @@
+LOADED_NPC_MAX EQU $08
+
+; wLoadedNPCs structure
+	const_def
+	const LOADED_NPC_ID
+	const LOADED_NPC_SPRITE
+	const LOADED_NPC_COORD_X
+	const LOADED_NPC_COORD_Y
+	const LOADED_NPC_DIRECTION
+	const LOADED_NPC_FIELD_05
+	const LOADED_NPC_FIELD_06
+	const LOADED_NPC_FIELD_07
+	const LOADED_NPC_FIELD_08
+	const LOADED_NPC_FIELD_09
+	const LOADED_NPC_FIELD_0A
+	const LOADED_NPC_FIELD_0B
+LOADED_NPC_LENGTH EQU const_value
```

Comment:
Would it make more sense to keep these values next to the definition of the `loaded_npc_struct` macro?

Reply:
like putting it in macros/wram.asm? I prefer constants being in /constants/ as much as possible to make it easier to find, but I'm open to changing my mind

Reply:
Yeah, in that file.
Generally I agree, but constants are usually enumerations of some kind.
These are just offsets that are only relevant to the struct.
I'm fine with it either way.